stacks-signer/src/tests/conf/signer-0.toml

```diff
@@ -2,3 +2,4 @@ stacks_private_key = "6a1fc1a3183018c6d79a4e11e154d2bdad2d89ac8bc1b0a021de8b4d28
 node_host = "127.0.0.1:20443"
 endpoint = "localhost:30000"
 network = "testnet"
+db_path = ":memory:"
```

Comment:
This should get deleted

Reply:
We made it mandatory again (I presume), so I've kept this and added the same to `signer-1.toml`. Let me know if you disagree with having this parameter required in the config.